src/flavors/__tests__/basic.test.ts

```diff
@@ -54,11 +54,13 @@ describe('flavors.basic', () => {
             nullableNestedValue: {
                 nullableStringValue: 'nested',
             },
+            // NB: This field is not transformed (b/c @Expose() is not included)
+            string_value_with_different_wire_name: 'value',
```

Comment:
Notice me: I'm assuming the exclusion of `expose` on `DecoratorOptions` was intentional, meaning that this now adds a new (expected) issue.

Reply:
Yeah, the `basic` flavor omits the `@Expose` decorator because we used to have a scenario in `pod` where that behavior broke things.